tests/preprocess/test_vhi.py

```diff
@@ -213,9 +213,66 @@ def test_alternative_region_interim_creation(self, tmp_path):
             regrid=regrid_dataset
         )
 
-        expected_out_path = tmp_path / 'interim/vhi_interim/\
+        expected_out_path = tmp_path / 'interim/VHI_interim/\
         STAR_VHP.G04.C07.NC_1981_8_31_ethiopia_VH.nc'.replace(' ', '')
         assert expected_out_path.exists(), \
             f'Expected processed file to be saved to {expected_out_path}'
         assert out == expected_out_path, f"Expected: {expected_out_path}, \
         Got: {out}"
+
+    def test_VCI(self, tmp_path):
+        v = VHIPreprocessor(tmp_path, var='VCI')
+
+        # get filename
+        demo_raw_folder = (v.raw_folder / 'vhi' / '1981')
+        demo_raw_folder.mkdir(parents=True, exist_ok=True)
+        netcdf_filepath = demo_raw_folder / 'VHP.G04.C07.NC.P1981035.VH.nc'
+
+        raw_height, raw_width = 360, 100
```

Comment:
is there not a `_make_dummy_vhi_dataset()` function? We could put all of these into a call to that funciton?

Reply:
```
    @staticmethod
    def _make_vhi_dataset():
        # build dummy .nc object
        height = list(range(0, 3616))
        width = list(range(0, 10000))
        vci = tci = vhi = np.random.randint(100, size=(3616, 10000))
        height = list(range(0, 3616 // 100))
        width = list(range(0, 10000 // 100))
        vci = tci = vhi = np.random.randint(100, size=(len(height), len(width)))

        ds = xr.Dataset(
            {'VCI': (['HEIGHT', 'WIDTH'], vci),
             'TCI': (['HEIGHT', 'WIDTH'], tci),
             'VHI': (['HEIGHT', 'WIDTH'], vhi)},
            coords={
                'HEIGHT': height,
                'WIDTH': width}
        )
        return ds

```

Reply:
yes! it was unused in all of the tests haha. Updated